examples/avro-cli.py

```diff
@@ -57,21 +76,34 @@ def to_dict(self):
         }
 
 
-# Handle delivery reports served from producer.poll.
-def on_delivery(err, msg):
+"""
+    Handle delivery reports served from producer.poll.
+"""
+
+
+def on_delivery(err, msg, request):
```

Comment:
suggest renaming `request` to `user` or `obj`, since that is what it is, it is not a request.